manifests/g/Geniatech/eyeTV/4/Geniatech.eyeTV.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Geniatech.eyeTV
+PackageVersion: 4
+PackageLocale: en-US
+Publisher: Geniatech
+PackageName: eyeTV 4

```

```suggestion
PackageName: EyeTV
```
This should match control panel as closely as possible